dev/Debian9_qt4/start_mxcube2

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env bash
-CONTAINER=mx3_deb9_qt4
```

Comment:
This is the name of the base docker image also used in the demo Debian9_qt4 image. You must either keep the original name or change also the name of the base image for the Debian9_qt4 demo image.

Reply:
Not sure I understand this one. For one thing, should the demo and the dev docker necessarily have the same setup? For another, where do I find "the base image `mx3_deb9_qt4` provided in this repository.", and the docker file that describes it?